### PR TITLE
fix import task

### PR DESCRIPTION
**Pull request by veeding**
don't need to pass postStart

### CODE REVIEW

**Review by hetao92 — APPROVED** [review]
LGTM

File: server-v2/api/studio/internal/service/import.go [hunk 1]
```diff
@@ -309,6 +309,13 @@ func validClientParams(conf *importconfig.YAMLConfig) error {
 		}
 	}
 
+	if conf.NebulaClientSettings.PostStart != nil {
+		if conf.NebulaClientSettings.PostStart == nil ||
+			*conf.NebulaClientSettings.PostStart.AfterPeriod == "" {
```
Comment (veezhang) [inline server-v2/api/studio/internal/service/import.go]: Could `AfterPeriod` be nil?

Comment (fengluodb) [reply server-v2/api/studio/internal/service/import.go:314]: it's ok

Comment (veeding) [reply server-v2/api/studio/internal/service/import.go]: It' ok

Comment (veezhang) [reply server-v2/api/studio/internal/service/import.go]: If `AfterPeriod` is nil, so `*conf.NebulaClientSettings.PostStart.AfterPeriod == ""` will panic.
At first, you set `PostStart` = nil, and now you set `PostStart.AfterPeriod = nil`, so why are you adjusting here?

**Review by veezhang — APPROVED** [review]
LGTM

**Pull request merged**